### PR TITLE
⚗️(frontend) show username on AccountDropDown

### PR DESCRIPTION
**Pull request by daproclaima**
## Purpose
Knowing who we are once logged in La Régie would be useful.

## Proposal

Show the full name of the currently connected user on the account drop down in the right top corner

### Before

https://github.com/user-attachments/assets/1655a0dd-4a03-4792-8a00-6bd7ac052b32

### After 

https://github.com/user-attachments/assets/050c10a4-a522-4870-a155-a3a159fae5e9

### CODE REVIEW

File: src/frontend/apps/desk/src/features/header/AccountDropdown.tsx [hunk 1]
```diff
@@ -7,32 +7,38 @@ import { useAuthStore } from '@/core/auth';
...
 export const AccountDropdown = () => {
   const { t } = useTranslation();
-  const { logout } = useAuthStore();
+  const { userData, logout } = useAuthStore();
 
+  const userName = userData?.name || t('No Username');
   return (
     <DropButton
-      aria-label={t('My account')}
+      aria-label={userName}
```
Comment (AntoLC) [inline src/frontend/apps/desk/src/features/header/AccountDropdown.tsx:15]: There is no `aria-label` prop on this component.

File: secrets
Comment (AntoLC) [inline secrets:1]: Like the other PR, there is the secrets versioned, rebase problem ?

**Review by AntoLC — COMMENTED** [review]
Not blocking for merging.
I noticed a strange focus after we close the dropdown, see below:
 
[scrnli_9_24_2024_9-54-49 AM.webm](https://github.com/user-attachments/assets/c6407caa-2047-4f91-ab47-351ee343b0d8)

**Comment by daproclaima** [comment]
> Not blocking for merging. I noticed a strange focus after we close the dropdown, see below:
> 
>  scrnli_9_24_2024_9-54-49.AM.webm

Yes, I noticed it too. It looks like this behavior is integrated into the Cunningham Dropwown component.

**Review by AntoLC — APPROVED** [review]

**Pull request merged**